src/backend/joanie/settings.py

```diff
@@ -497,9 +497,14 @@ class Build(Base):
     """
 
     SECRET_KEY = values.Value("DummyKey")
-    STORAGES_STATICFILES_BACKEND = values.Value(
-        "whitenoise.storage.CompressedManifestStaticFilesStorage"
-    )
+    STORAGES = {
+        "staticfiles": {
+            "BACKEND": values.Value(
+                "whitenoise.storage.CompressedManifestStaticFilesStorage",
+                environ_name="STORAGES_STATICFILES_BACKEND",
+            ),
+        }
+    }
```

Comment:
IMO there is no need to do that in this PR, it's out of the scope and it will be managed in PR #452

Reply:
In fact, `tray` job fails with Staging environment because of that ...